imageroot/actions/create-module/50start-bouncer

```diff
@@ -17,18 +17,17 @@ After=${MODULE_ID}.service
 
 [Service]
 Restart=always
-ExecStartPre=/usr/bin/bash -c "if ! ipset list crowdsec-blacklists > /dev/null; then firewall-cmd --permanent --new-ipset=crowdsec-blacklists --type=hash:ip --option="timeout=0" --option="maxelem=150000"; fi"
-ExecStartPre=/usr/bin/bash -c "if ! ipset list crowdsec6-blacklists > /dev/null; then firewall-cmd --permanent --new-ipset=crowdsec6-blacklists --option=family=inet6 --type=hash:ip --option="timeout=0" --option="maxelem=150000"; fi"
-ExecStartPre=firewall-cmd --reload
```

Comment:
IN the previous implementation you called `firewall-cmd --reload`, but the same command is not present inside `firewall-rules` script. Is that correct?

Reply:
we does it inside the script file

https://github.com/NethServer/ns8-crowdsec/pull/10/files#diff-30aeaf113aa5889f770cb8c5876e8d322d1910c16999d8aac147704f59ee19f5R16